Apps/Database/src/Context/GatewayDbContext.cs

```diff
@@ -73,8 +73,9 @@ public GatewayDbContext(DbContextOptions<GatewayDbContext> options)
         public DbSet<Communication> Communication { get; set; } = null!;
         public DbSet<CommunicationEmail> CommunicationEmail { get; set; } = null!;
         public DbSet<Rating> Rating { get; set; } = null!;
-        public DbSet<UserDelegate> UserDelegate { get; set; } = null!;
-        public DbSet<UserDelegateHistory> UserDelegateHistory { get; set; } = null!;
+        public DbSet<ResourceDelegate> ResourceDelegate { get; set; } = null!;
+        public DbSet<ResourceDelegateHistory> ResourceDelegateHistory { get; set; } = null!;
+        public DbSet<ResourceDelegateReasonCode> ResourceDelegateReasonCode { get; set; } = null!;
```

Comment:
NIT:  You don't need to explicitly call out code tables unless you want to be able to perform operations on them.

Reply:
good catch, I've removed them.